Makefile

```
@@ -155,7 +155,7 @@ test:
 
 test-smoke:
 	$(info log-file: test-$(TEST_SUFFIX).log)
-	PASSES="fmt build unit" ./test.sh 2<&1 | tee test-$(TEST_SUFFIX).log
+	PASSES="fmt bom dep build unit" ./test.sh 2<&1 | tee test-$(TEST_SUFFIX).log
```

This should force `make` to exit with a failure if one of the passes fails, as spotted by @jmhbnz.

```suggestion
	PASSES="fmt bom dep build unit" ./test.sh 2<&1 | tee test-$(TEST_SUFFIX).log
	! grep "FAIL:" test-$(TEST_SUFFIX).log
```

---

Ack, implemented the suggested fix, thanks. @ivanvc @jmhbnz